scheduler/scheduler_unit_test.go

```diff
@@ -356,8 +367,51 @@ func (suite *SchedulerTestSuite) TestSchdulerDriverStopOK() {
 
 	suite.False(driver.Stopped())
 	suite.Equal(mesos.Status_DRIVER_RUNNING, driver.Status())
+	driver.connected = true // pretend that we're already registered
```

Comment:
we don't really want a public method to set the driver state. it's definitely something that should be package-private.